native/src/layout/node.rs

```diff
@@ -54,7 +54,11 @@ impl Node {
         &self.children
     }
```

Comment:
Instead of breaking the existing methods, I would add a `move_to` method. Also, I think we should use `Point` instead of `Size` because we are not setting the `bound` of the `Node` but its `position`.

Therefore:

```rust
pub fn move_to(&mut self, position: Point) {
    self.bounds.x = position.x;
    self.bounds.y = position.y;
}
```

I think this is more explicit because having a `Node` not positioned at the origin is unusual.

Reply:
Sound much better, thanks.